web/src/routes/admin/system-settings/+page.svelte

```diff
@@ -180,6 +182,13 @@
       key: 'notifications',
       icon: mdiBellOutline,
     },
+    {
+      component: TemplateSettings,
```

Comment:
I wonder if the template settings is within the email settings would be more logical?

Reply:
Will there be other notification types in the future apart from emails? Then it seems logical to me to split it up, so templates are generic over any notifcation type?

Reply:
Other notification types might not support a full templated body like emails do. For now I think it makes sense to be under emails, in the future we can always move it if necessary.

Reply:
Ok, i'll change it :)